src/App.js

```diff
@@ -76,6 +80,40 @@ export default class App extends Component {
     this.setState({characterName: event.target.value})
   }
 
+  updateAbilityScore (ability, score) {
+    let newScores = this.state.activeCharacter.hasOwnProperty('abilityScores')
+      ? Object.assign(this.state.activeCharacter.abilityScores, {[ability]: score})
+      : Object.assign({STR: 8, DEX: 8, CON: 8, INT: 8, WIS: 8, CHA: 8}, {[ability]: score})
```

Comment:
ick. multi-line ternary operations are Evil Incarnate. At this point, just do an `if` ... the ternary isn't helpful here.